codec.proto

```diff
@@ -33,28 +31,28 @@ enum BlockIDFlag {
 
 // EventData
 message EventData {
-  Event     event = 1;
-  EventList block = 2;
+  Event      event = 1;
+  EventBlock block = 2;
```

Comment:
this seems like its not being rebased from Main. The current master already has EventBlock, can you rebase/merge it first so we can make sure we're looking at the same